components/molecules/Menu/MenuData.ts

```diff
@@ -18,6 +18,10 @@ export const menuData = [
     subMenu: [
       { title: "Verso por verso", href: "/estudios/verso-por-verso" },
       { title: "Estudios temáticos", href: "/estudios/estudios-tematicos" },
+      {
+        title: "Proverbios Imágenes",
```

Comment:
Buddy, I'm not sure, but I think that if you used couple lines before "Estudios temáticos" (second word not capitalized), you must use "Proverbios imágenes" here without "I" capitalized, just to keep the standard...  @SthefaniaEcheverry any thoughts?

Reply:
@DanielCalleCO  remove the capitalization of the word